OpenSim/Simulation/Model/Frame.h

```diff
@@ -185,6 +193,11 @@ OpenSim_DECLARE_ABSTRACT_OBJECT(Frame, ModelComponent);
     */
     SimTK::Transform findTransformInBaseFrame() const;
 
+    /** Accessor for position of Frame in Ground frame as output. */
```

Comment:
The comment should clarify that this means the origin of the frame. Also, Station has a method called `findLocation...`. But Model also already has a `com_position` output. After the hackathon, we should choose one and be consistent.